tests/baselines/reference/moduleAugmentationDeclarationEmit1.js

```diff
@@ -0,0 +1,68 @@
+//// [tests/cases/compiler/moduleAugmentationDeclarationEmit1.ts] ////
+
+//// [map.ts]
+
+import { Observable } from "./observable"
+
+(<any>Observable.prototype).map = function() { }
```

Comment:
our test suite is not a set of examples and I'd prefer not to pollute it with things that are non-relevant to feature being tested. Once we finalize the design and check-in the implementation we can add as many examples to the handbook as we want